Added LayerMicaUI Taskbar Theme

Nimai-HK opened the pull request:
##LayerMicaUI Taskbar theme

- Strictly for left aligned taskbar
- Widgets must be disabled

- Replaced legacy image icons of Bluetooth and Safely Remove Hardware Device, with Segoe Fluent Icons Glyph

- Taskbar Labels, Height and icon size, Spacing, Clock customisation are all to be done from their respective mods.

- Main elements use Acrylic with certain luminosity to mimic mica
- Background elements and some other surfaces use Windhawk Blur

- Works in both light and dark theme

<img width="640" height="135" alt="screenshot" src="https://github.com/user-attachments/assets/65ae194c-0662-4b45-90f4-a9961721ddda" />

Nimai-HK commented:
*Want to make an additional change, after the auto accent update is out. For now, I'll convert this PR to a draft.

m417z commented:
> I didn't know that the feature would arrive 2 updates further away.

That's not what I meant. The feature will be included in the next update. I meant that if you'll wait for the next mod update to be released before finishing your theme, then the theme itself won't be a part of that update.

Nimai-HK commented:
Theme completed.
- auto accent colourisation used for Bluetooth and safely remove hardware icons
- changed the z-index of above icons
- updated images, added an extra image hidden under clock customisation details in the README

SandTechStuff commented on Themes/LayerMicaUI/README.md:
````diff
@@ -0,0 +1,322 @@
+
+```json
+{
+  "controlStyles[0].styles[0]":"Background:=<WindhawkBlur BlurAmount=\"5\" TintColor=\"{ThemeResource Chrome}\" TintOpacity=\"0.3\"  />",
````
`{ThemeResource Chrome}` is not a valid ThemeResource afaik, try something like `{ThemeResource SystemChromeHighColor}`

SandTechStuff reviewed (approved):
Looks good, thanks for the submission! 

@m417z You can merge if you feel it is ready as well, or would it be better to wait until the new taskbar styler features are added to the marketplace?

m417z commented:
I added a note about it. I'll remove it once the new version is released.
